submissions/MosQuitO404/js-dom/index.js

```diff
@@ -0,0 +1,129 @@
+
+    williamsInfo: '<h3>In profile</h3><p>Driven on by the brilliance and passion of the late Sir Frank Williams, Williams grew from humble beginnings to become a Formula 1 behemoth, unrivalled by all except Ferrari and McLaren in terms of enduring success. Over the past four decades the team has racked up Grand Prix wins and championship glory, and in the process nurtured some of the greatest talents in the sport, both in and out the cockpit. Now, following the Williams family`s decision to step aside after the sale of the team to Dorilton Capital, a new era begins...</p>'
+}
+
+// Create teams
+
+class Team {
+    #teamTitle
+    #firstPilotName;
+    #secondPilotName;
+    #firstPilotFlagImage;
+    #secondPilotFlagImage;
+    #firstPilotImage;
+    #secondPilotImage;
+    #teamInfo;
+    #html;
+
+    constructor(teamTitle, firstPilotName, secondPilotName, firstPilotFlagImage, secondPilotFlagImage, firstPilotImage, secondPilotImage, teamInfo) {
+        this.#teamTitle = teamTitle;
+        this.#firstPilotName = firstPilotName;
+        this.#secondPilotName = secondPilotName;
+        this.#firstPilotFlagImage = firstPilotFlagImage;
+        this.#secondPilotFlagImage = secondPilotFlagImage;
+        this.#firstPilotImage = firstPilotImage;
+        this.#secondPilotImage = secondPilotImage;
+        this.#teamInfo = teamInfo;
+        this.#html = `<div class="pilots-container">
+                    <div class="pilot-wrapper">
+                        <span id="first-pilot-name" class="pilot-name"><h3>${this.#firstPilotName}</h3></span>
+                        <span id="first-pilot-flag"><img class="flag-img" src="${this.#firstPilotFlagImage}" alt=""></span>
+                        <img id="first-pilot" class="pilot-img" src="${this.#firstPilotImage}" alt="">
+                    </div>
+                    <div class="pilot-wrapper">
+                        <span id="second-pilot-name" class="pilot-name"><h3>${this.#secondPilotName}</h3></span>
+                        <span id="second-pilot-flag"><img class="flag-img" src="${this.#secondPilotFlagImage}" alt=""></span>
+                        <img id="second-pilot" class="pilot-img" src="${this.#secondPilotImage}" alt="">
+                    </div>
+                    <div class="team-info-container">${this.#teamInfo}</div>
+                </div>`
+    }
+
+    get htmlRender() {
```

Comment:
With this type of architecture, you add eventListener on `menu` and `popupMenu` every time when you initialise Team. Try to figure out a solution when you init eventListeners only once.

Reply:
@lysenko-sergey-developer I don't know how to do this, I need a hint, please :(

Reply:
Let's try to do it step by step.
First of all, I should know do you delete Team class or solved problem with long params?

Reply:
@lysenko-sergey-developer I didn't delete Team class and solved problem with long parameters. And think I resolved the problem with eventListener too :) It's looking like a very strange decision, but you'll see.

Reply:
Commit your current working version I will look to code.

Reply:
@lysenko-sergey-developer done, look please. This current version have working eventListener and creates exemplars of class. Next step - working with DOM and html rendering, but I haven't finished it yet.

Reply:
Committed working version (all bugs resolved)